common/lib/driver-definitions/src/storage.ts

```diff
@@ -304,7 +304,10 @@ export interface IDocumentServiceFactory {
     /**
      * Returns an instance of IDocumentService
      */
-    createDocumentService(resolvedUrl: IResolvedUrl, logger?: ITelemetryBaseLogger): Promise<IDocumentService>;
+    createDocumentService(
+        resolvedUrl: IResolvedUrl,
+        clientType: string,
```

Comment:
I'd prefer this to be a boolean (if you want to use strings, then I'd expect this layer to define all the types here in the form of enum or string union).